.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,44 @@
+---
+name: Bug Report
+about: Create a report to help us improve
+title: ''
+labels: bug
+assignees: ''
+
+---
+
+**Describe the Bug**
+A clear and concise description of what the bug is.
+
+**To Reproduce**
+Steps to reproduce the behavior:
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected Behavior**
+A clear and concise description of what you expected to happen.
+
+**Actual Behavior**
+A clear and concise description of what actually happens.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (Please Complete the Following Information)**
```

Comment:
I would omit the "(Please Complete ...)", same goes for the "Smartphone" heading.

Reply:
-> 97de6f839a26c4953b901b7a1db368516eff21c6